solutions/logging/core-env/org-sink.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+######
+# Organization sink for Security logs: Cloud Audit, Access Transparency, and Data Access Logs
+# Destination: Cloud Logging bucket hosted inside logging project
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
+kind: LoggingLogSink
+metadata:
+  name: logging-prj-id-security-sink # kpt-set: ${logging-prj-id}-security-sink
+  namespace: logging
+  annotations:
+    config.kubernetes.io/depends-on: security-log-bucket # kpt-set: logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/${security-log-bucket}
+spec:
+  organizationRef:
+    external: "0000000000" # kpt-set: ${org-id}
+  includeChildren: true
+  destination:
+    loggingLogBucketRef:
+      # destination.loggingLogBucketRef
+      # Only `external` field is supported to configure the reference.
+      external: security-log-bucket # kpt-set: logging.googleapis.com/projects/${logging-prj-id}/locations/northamerica-northeast1/buckets/${security-log-bucket}
+  description: Organization sink for Security Logs
+  # AU-2, AU-12(A), AU-12(C)
+  # Includes Security logs: Cloud Audit, Access Transparency, and Data Access Logs
+  # Security logs help you answer "who did what, where, and when"
+  #
+  # Cloud Audit Logs:
+  #  Admin Activity
+  #  Data Access
+  #  System Events
+  #  Policy Denied
+  #
+  # Access Transparency Logs (TODO - not enabled)
```

Comment:
very minor thing but my VSCode extension highlights in yellow `TODO:` but not `TODO`. are you using that extension ? maybe it can be customized. I'd have to check

Reply:
It might be your extension - does not highlight on my end